VivServer enhancements (#615)
vivisect/vivisect: docs/vivisect/quickstart.rst

    vivbin /path/to/my/bulk/binary.exe.viv

Collaborative Reverse Engineering
=================================

> One of the most powerful aspects of Vivisect is it's real-time collaboration server capabilities.  There are two ways to collaborate with others over a network.

Author: ```suggestion
One of the most powerful aspects of Vivisect is its real-time collaboration server capabilities.  There are two ways to collaborate with others over a network.
```

- The first way is completely from the GUI, choosing `Share->Share Workspace` from one instance of Vivisect (vivbin), and others choosing `Share->Connect to Shared Workspace` from a separate Vivisect instance

> - The other way to collaborate over a network (localhost works too) is to run a VivServer.  With a VivServer, you can share an entire directory structure of VivWorkspace files.

Author: ```suggestion
- The other way to collaborate over a network, or using localhost, is to run a VivServer.  With a VivServer, you can share an entire directory structure of VivWorkspace files.
```

> Because of the event-driven Vivisect storage model, actions taken in one Vivisect window is easily redirected to the remote Server/Workspace, and other connected workspaces updated within seconds.

Author: done

Running the VivServer
=====================

Running the VivServer is simple::

    vivserver /path/to/top/directory

Interacting with the VivServer
==============================

Saving a workspace to a VivServer is as simple as `File->Save to Server`

> A popup lets you choose the server and the path.  A checkbox lets you set the default server.  If you access one server the majority of the time, you will find this very helpful.  If you want to access another server once or twice, don't check this box, and your default won't be affected

Author: ```suggestion
A popup lets you choose the server and the path.  A checkbox lets you set the default server.  If you access one server the majority of the time, you will find this very helpful.  If you want to access another server once or twice, don't check this box, and your default won't be affected.
```

> The FilePath is where in the remote file repository this workspace will be stored, and it's name.  (Be sure to add the .viv extension to the filename)

Author: ```suggestion
The FilePath is where in the remote file repository this workspace will be stored, and it's name (Be sure to add the .viv extension to the filename).
```

> The FilePath is where in the remote file repository this workspace will be stored, and it's name.  (Be sure to add the .viv extension to the filename)

Author: ```suggestion
The FilePath is where in the remote file repository this workspace will be stored, and its name.  (Be sure to add the .viv extension to the filename)
```
